cosmovisor/.goreleaser.yml

```diff
@@ -6,12 +6,10 @@ release:
 
 before:
   hooks:
-    - go mod tidy
+    # - go mod tidy
```

Comment:
I think this will break until https://github.com/cosmos/cosmos-sdk/pull/12851#discussion_r941885873 - As we reverted the go.work changes for fixing `make test`.